src/component/FormFields/Attribute.js

```diff
@@ -8,12 +8,15 @@ import data from '@groceristar/groceristar-fetch/search';
 let attribute;
 
 function toOpt(arr) {
-    let Opt= arr.reduce((intermediate, item, index) => {
+    let Opt
+    if(typeof arr === 'object') {
+        Opt = arr.reduce((intermediate, item, index) => {
```

Comment:
what do you think about installing lodash and using it for this type of functions? 
i think it'll make it less complicated

Reply:
Well, I haven't used lodash so I'll have to look it up at the documentation.

Reply:
sure, no prob. you'll like it.
i'll merge this PR, but later when you'll have time - check that lib. will create a separated issue for it

Reply:
Sure! :)